lib/util.js

```diff
@@ -423,3 +423,21 @@ function compareByGeneratedPositionsInflated(mappingA, mappingB) {
   return strcmp(mappingA.name, mappingB.name);
 }
 exports.compareByGeneratedPositionsInflated = compareByGeneratedPositionsInflated;
+
+/**
+ * Strip any JSON XSSI avoidance prefix from the string (as documented
+ * in the source maps specification), and then parse the string as
+ * JSON.
+ */
+function parseJSON(str) {
+  if (str.startsWith(")]}'")) {
+    var index = str.indexOf("\n");
+    if (index < 0) {
+      // No line, return nothing.
+      return null;
```

Comment:
I'd rather throw an error, just like JSON.parse throws an error if there is no input.